includes/modules/ezpages_bar_footer.php

```diff
@@ -17,14 +17,20 @@
 
 // test if bar should display:
 if (EZPAGES_STATUS_FOOTER == '1' || (EZPAGES_STATUS_FOOTER == '2' && (strstr(EXCLUDE_ADMIN_IP_FOR_MAINTENANCE, $_SERVER['REMOTE_ADDR'])))) {
-  $pages_query = $db->Execute("SELECT e.pages_id, e.page_open_new_window, e.page_is_ssl, e.alt_url, e.alt_url_external, e.toc_chapter, ec.pages_title
+
+  if (defined('TABLE_EZPAGES_CONTENT')) { 
```

Comment:
My concern here is that the code begins to assume that the normally-expected dependencies do NOT exist, which feels backwards.